core/templates/services/svg-sanitizer.service.ts

```diff
@@ -248,23 +251,27 @@ export class SvgSanitizerService {
 
   getIssueURL(
       invalidTagsAndAttributes: { tags: string[]; attrs: string[] }): string {
-    const baseURL = 'https://github.com/oppia/oppia/issues/new?title=Uploaded%20SVG%20image%20looks%20distorted%20in%20the%20preview&body=The%20image%20file%20is%20attached%20below:%0A%0A%7B%7BIMAGE_HERE%7D%7D%0A%0AScreenshots%20of%20the%20problem:%0A%0A%7B%7BSCREENSHOTS_HERE%7D%7D%0A%0AThe%20invalid%20tags%20and%20attributes%20were:';
-    let updatedURL = baseURL;
-    let invalidTags = invalidTagsAndAttributes.tags;
-    let invalidAttributes = invalidTagsAndAttributes.attrs;
+    const baseURL = 'https://github.com/oppia/oppia/issues/new?title=Uploaded%20SVG%20image%20looks%20distorted%20in%20the%20preview&body=The%20image%20file%20is%20attached%20below%3A%0A%0A%7B%7BIMAGE_HERE%7D%7D%0A%0AScreenshots%20of%20the%20problem%3A%0A%0A%7B%7BSCREENSHOTS_HERE%7D%7D%0A%0AThe%20invalid%20tags%20and%20attributes%20were%3A';
+    let uncodedURL = '';
```

Comment:
unencoded, not uncoded

Also perhaps you might want to start with the unencoded base URL. You can perhaps do it like this:
- Have a string that represents the issueMessage. Write that as regular text.
- Then create the URL as 'https://github.com/oppia/oppia/issues/new?title=' + encodedUriComponent(issueMessage)

That way, all your raw strings are human-readable.

Reply:
I have updated the method to make all raw strings human-readable PTAL. Thanks!